packaging/hudi-flink-bundle/pom.xml

```diff
@@ -289,6 +289,10 @@
                   <shadedPattern>org.apache.hudi.org.apache.hadoop.metrics2.util.MetricSampleQuantiles
                   </shadedPattern>
                 </relocation>
+                <relocation>
+                  <pattern>org.apache.hadoop.hive.serde2.</pattern>
+                  <shadedPattern>${flink.bundle.shade.prefix}org.apache.hadoop.hive.serde2.</shadedPattern>
+                </relocation>
```

Comment:
@gaogao110 from the ticket it reports as a problem in 0.10.1. Does this also fix master version ?